File: ALEX/convertidor.c
#include <stdio.h>
#include <math.h>
#define cambio 1.17


int bienvenida()
{
	printf("Bienvenido a  CONVERTIDOR\n");
	printf("by Alex\n");
	printf("Version 2.0\n");
	printf ("by Alex FerLO");
	return (0);
}
float convertidor()
{
	float EURO;
	EURO = 0.0;
	float DOLAR;
	printf("EURO\n");
	scanf("%f",&EURO);
	DOLAR = EURO * cambio;
	printf ("%f\n",DOLAR);
	printf("DOLAR\n");
	return (DOLAR);
}
int despedida()
{
	printf("Gracias por utilizar  CONVERTIDOR\n");
}
int main()
{	
	bienvenida ();
	while (0==0)
	{
		convertidor();
	}
	despedida();
	return (0);
}
	
	

File: ALEX/Java/convertidor/src/main/java/convertidor.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author alex
 */
import java.util.Scanner;
public class convertidor {
   public static void main(String[] args) {
        // TODO code application logic here
        Scanner in = new Scanner(System.in);
    
	System.out.println("Bienvenido a  CONVERTIDOR");
	System.out.println("by Alex");
	System.out.println("Version 3.0");
	System.out.println("by Alex FerLO");
	
            System.out.println("DOLAR : ");
            double DOLAR = in.nextInt();
            double cambio = DOLAR*1.33250;
            System.out.println(cambio + " €");
       System.out.println("Gracias por utilizar  CONVERTIDOR");
}
}

	